vaadin-dialog-flow-parent/vaadin-dialog-flow/src/main/java/com/vaadin/flow/component/dialog/Dialog.java

```diff
@@ -179,6 +188,22 @@ public String getHeight() {
         }
     }
 
+    /**
+     * `opened-changed` event is sent when the overlay opened state changes.
```

Comment:
Backtick notation doesn't work correctly in JavaDoc. See a [similar comment in context-menu base](https://github.com/vaadin/flow-components/blob/master/vaadin-context-menu-flow-parent/vaadin-context-menu-flow/src/main/java/com/vaadin/flow/component/contextmenu/ContextMenuBase.java#L313).

<img width="698" alt="Screenshot 2022-12-09 at 12 33 34" src="https://user-images.githubusercontent.com/1222264/206683015-56c939f1-6d29-456d-ad3e-0adc89a31504.png">

Reply:
Done.